CONTRIBUTING.md

```diff
@@ -10,22 +10,22 @@ Contributors Guide
 Discuss contributions to DC/OS SDK using GitHub [issues](https://github.com/mesosphere/dcos-commons/issues) and make contributions with GitHub [pull requests](https://github.com/mesosphere/dcos-commons/pulls).
 
 ### Proposing Features
-If you would like to propose a new feature or a significant change, first open an issue to discuss it.
+If you would like to propose a new feature or a significant change, first open an issue to discuss it.  Project shepherds will provide feedback on the proposed change in terms of feasability, design, and implementation guidance.  While all proposals are welcome, those that conflict with our roadmap or design principles may not be accepted.
```

Comment:
Github appears to intelligently render only one space, so I'll leave it as is for now.